kiwi/builder/install.py

```diff
@@ -388,6 +388,14 @@ def _create_pxe_install_kernel_and_initrd(self):
                 self.boot_image_task.omit_module(
                     'multipath', install_media=True
                 )
+            for module in self.xml_state.get_installinitrd_modules('add'):
```

Comment:
The `self.xml_state.get_installationinitrd_modules` always returns a list (an empty one of no module is defined in XML) so it can be easily typed and the result is always iterable.

Reply:
got it